### DISCUSSION TITLE
Application for a technical writer

### DISCUSSION THREAD

**Discussion by ekasnh** [STEllAR-GROUP/hpx#5337] [Q&A] [answered]
Greetings of the day 
Hi my name is Ekansh Agarwal. I am pursuing Electronics and telecommunication Engineering. I would like to contribute to the project as a technical writer under this year's Google Season of Docs. Requesting your guidance for the same.

Regards
Ekansh Agarwal

**Comment by msimberg** [comment, marked as answer]
Hi @ekasnh, unfortunately the role has already been filled (the deadline for applying was last week). Wishing you better luck next year.

**Comment by ekasnh** [comment]
Greetings, thank you so much for notifying me
